RFC-003-SWAP-Basic.md

```diff
@@ -183,6 +185,43 @@ When Bob learns the secret from Alice's redeem activation of β-HTLC he MUST act
 He MUST make sure he does this before `alpha_expiry` or risks both losing **B** and not gaining **A**.
 To activate the redeem path he uses the secret and the procedure defined in the specification of the α-HTLC.
 
+## Expiry Time Considerations
+
+Conceptually there is a time window from when the HTLC is deployed until the HTLC expires and can be refunded.
+The protocol depends on the alpha time window being a superset of the beta time window.
+
+
```

Comment:
📣   I think you want to make this code-block. Please add backticks ``` before and after.

Reply:
Awesome, thanks.  Implemented as suggested.